samples/java_layout/build.gradle

```diff
@@ -20,8 +20,13 @@ android {
         versionCode 1
         versionName "1.0"
 
+        String universalDeepLinkHost = "www.java-sample.com"
```

Comment:
Since we do not own this domain name, Android will not be able to do auto verification for the app links and therefore, I expect that the web browser will open deep link URL or show a share dialog before opening our app? 

With the goal being this app should allow QA to test that our SDK works with Android app links, it's important that the URLs used by QA can be auto verifiable by Android and skip opening the browser and showing a dialog when opening a URL. 

Details on a domain name that is auto verifiable by our apps [is mentioned in slack](https://customerio.slack.com/archives/C01QYDKD0SU/p1683807088350899).

Reply:
The firebase dynamic link is a good solution, but this would work too. In Android, you can go to App settings and there you can find these links there once you allow them from settings, next time whenever that link is opened it will open your app rather than the browser, so QA should be able to test it.